library/src/main/java/io/github/luizgrp/sectionedrecyclerviewadapter/SectionedRecyclerViewAdapter.java

```diff
@@ -405,6 +424,33 @@ public int getSectionPosition(String tag) {
     }
 
     /**
+     * Return the section position in the adapter.
+     *
+     * @param section This section
+     * @return position of the section in the adapter
+     */
+    public int getSectionPosition(Section section) {
```

Comment:
Hi @yccheok,

Could you please create a PR to merge into `develop` branch including only this new method?